packages/enzyme-test-suite/test/shared/methods/setContext.jsx

```diff
@@ -112,7 +112,7 @@ export default function describeSetContext({
 
       expect(spy.args).to.deep.equal([
         ['render'],
-        ['componentWillReceiveProps'],
+        ...(is('>= 17') && isShallow ? [] : [['componentWillReceiveProps']]),
```

Comment:
I'm not sure why `isShallow` is necessary here. Is `shallow` broken, or have `componentWillReceiveProps` and `UNSAFE_componentWillReceiveProps` been removed in 17.x as promised here: https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops ?